### PR TITLE
Add Jay-Prakash to Before I Die

### PR DESCRIPTION
**Pull request by jayprakash25**
### Checklist

- [x] I have thoroughly read and understand [The Contribution Guidelines](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md).
- [ ] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behavior.
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc.
  - `Fix` - bug fixes.
  - `Docs` - fixing typos and adding new content.
- [x] I have listed my proposed changes and their benefits.
- [ ] I have linked the corresponding issue in the Issues section.

### CODE REVIEW

**Comment by XanderRubio** [comment]
Hi @jayprakash25! I'm currently traveling and once settled into my next location I will review and merge your contribution! Thank you for your time in contributing and I'm looking forward to reviewing. Have a nice day!

File: src/Jsons/Contributors.json
Comment (XanderRubio) [inline src/Jsons/Contributors.json:1]: Hi @jayprakash25,

Great work! Everything was done correctly, but your avatar didn't display due to a naming issue in the file path. Good to go!

**Comment by jayprakash25** [comment]
Hello @XanderRubio any updates about my pr

File: src/Jsons/Contributors.json
Comment (jayprakash25) [reply src/Jsons/Contributors.json:1]: Hi @XanderRubio . 
Thank you for changing it for me. Is my PR accepeted?

**Review by XanderRubio — APPROVED** [review]
@jayprakash25 Great work, and your text and image look great on the site. Live is what we all aim to do, and continue to make it happen. I like the photo you selected. Going ahead with merging now!

**Comment by XanderRubio** [comment]
@jayprakash25  Your pull request has been successfully merged!!! Congratulations Bottle with Popping CorkBottle with Popping CorkBottle with Popping CorkDouble Exclamation MarkDouble Exclamation MarkDouble Exclamation Mark. 


 If you haven't already, please go ahead and star Glowing StarGlowing StarGlowing Starthis repo and you are more than welcome to continually contribute to this project by assisting with our [open issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues), or adding your own new open issues, and additionally you can get an idea of ideas we have to continually grow this project further as technology and tech stack continually evolve by looking at the [ROADMAP](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/ROADMAP.md) for the project. Have a great day, and thank you for taking the time to contribute, 
Jay! You can view your contribution now on the [live link](https://before-i-die-achievements.vercel.app/). Have a great rest of your day, and I will be sending you an invite to be a member of the Before I Die Code now.

**Comment by jayprakash25** [comment]
Sure!!

File: src/Jsons/Contributors.json
Comment (XanderRubio) [reply src/Jsons/Contributors.json:1]: Yes, you PR has been merged.

**Pull request merged**